### ISSUE TITLE
ID 8-10 samples for EpiRAD ddRAD

### ISSUE THREAD

**Issue by sr320** [hputnam/project_larval_geoduck_OA#2]
Jay can possibly process a set of samples the week of January 23 with his corals.

Are there about 10 samples that would be worth running (ranked in order of priority)?

**Comment by hputnam** [comment]
Yes, will confirm and prep box of samples before I leave on Jan 7.
